FBG_Gotha_Chart_A_707_Descartes_duplicated_new_bundle.xml

```diff
@@ -533,6 +535,15 @@
 <lb ed="#P" n="2"/>und was eines ieden Umbkreises Krafft sey?
 <lb ed="#P" n="3"/>(quae sit cuiusqe sphaera activitatis.)
           </p>
+         <p><figure>
```

Comment:
don't forget to add the line number on which this figure occurs, as an `lb@type="fixed=pagid"` directly before the figure.

`lb@type="fixed" should also go on the next line, to let us know that the previous number sequence continues